src/coreclr/tools/aot/ilc.sln

```diff
@@ -242,30 +242,30 @@ Global
-		{8A189216-D635-3531-861A-C34679A4E081}.Release|Any CPU.ActiveCfg = Release|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.Release|Any CPU.Build.0 = Release|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.Release|x64.ActiveCfg = Release|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.Release|x64.Build.0 = Release|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.Release|x86.ActiveCfg = Release|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.Release|x86.Build.0 = Release|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.RelWithDebInfo|Any CPU.ActiveCfg = RelWithDebInfo|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.RelWithDebInfo|Any CPU.Build.0 = RelWithDebInfo|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.RelWithDebInfo|x64.ActiveCfg = RelWithDebInfo|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.RelWithDebInfo|x64.Build.0 = RelWithDebInfo|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.RelWithDebInfo|x86.ActiveCfg = RelWithDebInfo|x64
-		{8A189216-D635-3531-861A-C34679A4E081}.RelWithDebInfo|x86.Build.0 = RelWithDebInfo|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Checked|Any CPU.ActiveCfg = Checked|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Checked|Any CPU.Build.0 = Checked|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Checked|x64.ActiveCfg = Checked|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Checked|x64.Build.0 = Checked|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Checked|x86.ActiveCfg = Checked|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Checked|x86.Build.0 = Checked|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Debug|Any CPU.ActiveCfg = Debug|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Debug|Any CPU.Build.0 = Debug|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Debug|x64.ActiveCfg = Debug|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Debug|x64.Build.0 = Debug|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Debug|x86.ActiveCfg = Debug|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Debug|x86.Build.0 = Debug|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Release|Any CPU.ActiveCfg = Release|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Release|Any CPU.Build.0 = Release|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Release|x64.ActiveCfg = Release|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Release|x64.Build.0 = Release|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Release|x86.ActiveCfg = Release|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.Release|x86.Build.0 = Release|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.RelWithDebInfo|Any CPU.ActiveCfg = RelWithDebInfo|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.RelWithDebInfo|Any CPU.Build.0 = RelWithDebInfo|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.RelWithDebInfo|x64.ActiveCfg = RelWithDebInfo|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.RelWithDebInfo|x64.Build.0 = RelWithDebInfo|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.RelWithDebInfo|x86.ActiveCfg = RelWithDebInfo|x64
+		{AE4FFE05-5F31-31CF-84DC-4DD333FF3D22}.RelWithDebInfo|x86.Build.0 = RelWithDebInfo|x64
```

Comment:
`.sln` changes not reverted intentionally?

Reply:
Reverted